Finder.php

```diff
@@ -134,7 +134,20 @@ public function in($path)
         }
 
         foreach ($path as $p) {
-            $this->_paths[] = $p;
+            if (1 === preg_match('/[\*\?\[\]]/', $p)) {
+                $iterator = new Iterator\CallbackFilter(
+                    new Iterator\Glob(rtrim($p, DS)),
+                    function ($current) {
+                        return $current->isDir();
+                    }
+                );
+
+                foreach ($iterator as $path => $_fileInfo) {
```

Comment:
Actually, I would go with this solution:
```diff
- foreach ($iterator as $path => $_fileInfo) {
-     $this->_paths[] = $path;
+ foreach ($iterator as $fileInfo) {
+     $this->_paths[] = $fileInfo->getPathname();
```
Why? Because the key from the iterator can change, and it will break this part of the code. So this is error-prone. Moreover, you will resolve the variable name collision.